<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VesselEditorMainWindow</class>
 <widget class="QMainWindow" name="VesselEditorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Objects Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabPaths">
       <attribute name="title">
        <string>Paths</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Application Name</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="applicationName"/>
           </item>
           <item row="8" column="3">
            <widget class="QPushButton" name="browseGISPath">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="gisPath"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Application Path</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>iGraph Parameter value</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="iGraph"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Raw Data Path</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="browseInputPath">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="inputPath"/>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="browseBasePath">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Scripts base path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="scriptsPath"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>346</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVessels">
       <attribute name="title">
        <string>Vessels Specifications</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CsvSpecsPage" name="vesselsCsvPage" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabConfig">
       <attribute name="title">
        <string>Vessels Scripts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="genConfig">
            <property name="text">
             <string>Generate Config Files</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="run">
            <property name="text">
             <string>Generate Data File</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="genMetVar">
            <property name="text">
             <string>Generate Metiers Files</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="genMetSelectivity">
            <property name="text">
             <string>Generate Metiers Selectivity per Stock Files</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="log">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Populations Specs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTabWidget" name="popSpecsTab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Specs 1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="CsvSpecsPage" name="popSpecs1" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Specs 2</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="CsvSpecsPage" name="popSpecs2" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Specs 3</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="CsvSpecsPage" name="popSpecs3" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_Vessels_Spec">
   <property name="text">
    <string>&amp;Load Vessels Spec...</string>
   </property>
  </action>
  <action name="actionSave_Vessels_Spec">
   <property name="text">
    <string>Save Vessels Spec...</string>
   </property>
  </action>
  <action name="actionRscript_location">
   <property name="text">
    <string>Rscript executable location...</string>
   </property>
  </action>
  <action name="actionScripts_location">
   <property name="text">
    <string>Scripts Selection...</string>
   </property>
   <property name="toolTip">
    <string>Customize the scripts selection.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CsvSpecsPage</class>
   <extends>QWidget</extends>
   <header>csvspecspage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>genConfig</tabstop>
  <tabstop>run</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
